js-exercises/counting-anagrams/countingAnagrams.js

```diff
@@ -0,0 +1,26 @@
+const countingAnagrams = (str) => {
+    const tempAnagrams = {}
+    let flag = 0
+    str.forEach((loopingArrayWord)=>{
+      const wordAfterSorting = returnTheWordInSorted(loopingArrayWord)
```

Comment:
`returnTheWordInSorted()` is used before it was defined. Don't depend on `hoisting`. It's a bad practice.